Blue-Square-Legion/Keito-Yarn-Game
# [Pull Request] Star Ratings

## Changes

- Save different times for each level
- Clean up/remove PlayerPref bestTime
- Create Star Rating for each level
- Create Star UI
- Added new font that prints stars ★☆☆ properly

### Known Bugs/Issues
If the expected time to complete is exceeded, the game will print 0 stars even if the level is completed. This will be resolved once a game over timer is added in https://github.com/Blue-Square-Legion/Keito-Yarn-Game/issues/28

## Test Procedure

1. Build and run the game
2. Complete any level
Expected Result: Star Rating is shown for current time completion and best time completion.

## Issue ticket number/link
(https://github.com/Blue-Square-Legion/Keito-Yarn-Game/issues/29)

## Checklist before requesting a review
- [x] Make sure I am merging my features into the `develop` branch
- [x] I have notified the other members of the programming team that my task is ready to review
